test/typed_buffers_test.dart

```diff
@@ -185,13 +185,12 @@ void testInt(int bits, buffer(int length), {String testOn}) {
 }
 
 const List<int> intSamples = const [
-  0x10000000000000001,
-  0x10000000000000000, // 2^64
-  0x0ffffffffffffffff,
-  0xaaaaaaaaaaaaaaaa,
-  0x8000000000000001,
```

Comment:
The 16-digit hex numerals (that is, all but the first two numerals) are all valid Dart 2 integer literals.
Hex literals are allowed to have values in the range -2^63 .. 2^64-1, with the upper ones being normalized to negative values. This allows you to write *bit patterns* efficiently.
So, only remove the first two, 0x10000000000000001 and 0x10000000000000000.

Reply:
Done.